examples/shell/cmd_otcli.cpp

```diff
@@ -160,6 +160,13 @@ static int OnOtCliOutput(const char * aBuf, uint16_t aBufLength, void * aContext
 }
 #endif
 
+#if CHIP_TARGET_STYLE_EMBEDDED
+static otInstance * GetOtInstance()
```

Comment:
We have a merge conflict here - I replaced usage of this function with a direct reference to `otInstanceInitSingle`. As a result nRF Connect shell fails to build due to the function being unused. You need to remove it or use below instead of `otInstanceInitSingle`.

Reply:
Resolved